plugins/plugin_package/plugin_package.py

```diff
@@ -24,40 +26,19 @@ def brief_description():
         return MultiLanguage.get_string('PACKAGE_BRIEF')
 
     def parse_args(self, argv):
-        if len(argv) < 1:
-            self.print_help()
-            return None
-
         return {"command": argv[0]}
 
     def run(self, argv, dependencies):
-        args = self.parse_args(argv)
-        if args is None:
-            return
-
-        command = args["command"]
-
-        if command == "search":
-            from package_search import FrameworkAdd
-            CommandClass = FrameworkAdd
-        elif command == "info":
-            from package_info import PackageInfo
-            CommandClass = PackageInfo
-        elif command == "install":
-            from package_install import PackageInstall
-            CommandClass = PackageInstall
-        elif command == "list":
-            from package_list import PackageList
-            CommandClass = PackageList
-        elif command == "-h":
-            self.print_help()
-            return
-        else:
-            message = MultiLanguage.get_string('PACKAGE_ERROR_INVALID_CMD_FMT', command)
-            raise cocos.CCPluginError(message, cocos.CCPluginError.ERROR_CMD_NOT_FOUND)
-
-        commandObject = CommandClass()
-        commandObject.run(argv[1:])
+        cmd = self._get_sdkbox_path() + ' --runincocos ' + ' '.join(argv)
+        self._run_cmd(cmd)
+
+    def _run_cmd(self, command, cwd=None):
+        # cocos.CMDRunner.run_cmd(command, False, cwd=cwd)
+        subprocess.call(command, shell=True, cwd=cwd)
+
+    def _get_sdkbox_path(self):
+        path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Please get the path of current file like this:
```
    if getattr(sys, 'frozen', None):
        ret = os.path.realpath(os.path.dirname(sys.executable))
    else:
        ret = os.path.realpath(os.path.dirname(__file__))
```

It's used for maintaining compatibility with the situation: Building python scripts to binary with `pyinstaller`.